src/resource/ResourceManager.cpp

```diff
@@ -23,6 +23,7 @@ void ResourceManager::Init(const std::vector<std::string>& otrFiles, const std::
     // the extra `- 1` is because we reserve an extra thread for spdlog
     size_t threadCount = std::max(1, (int32_t)(std::thread::hardware_concurrency() - reservedThreadCount - 1));
     mThreadPool = std::make_shared<BS::thread_pool>(threadCount);
+    mAltAssetsEnabled = CVarGetInteger(CVAR_ALT_ASSETS, 0);
```

Comment:
Since you are making a dedicated `SetAltAssetsEnabled` method now, I think we could just drop `CVAR_ALT_ASSETS` entirely from the LUS repo and just have ports set the initial value on boot.

Reply:
You mean remove that line and have it called from the port on init?

Reply:
yea, I mean you could explicitly set to false for clarity, but I think it starts as false anyways.

Reply:
I declared it with false in the header.